v2/internal/controllers/alertsmanagement_smartdetectoralertrules_crud_v20210401_test.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	alertsmanagement "github.com/Azure/azure-service-operator/v2/api/alertsmanagement/v1api20210401"
+	insights "github.com/Azure/azure-service-operator/v2/api/insights/v1api20200202"
+	insightsag "github.com/Azure/azure-service-operator/v2/api/insights/v1api20230101"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+func Test_AlertsManagement_SmartDetectorAlertRules_CRUD(t *testing.T) {
```

Comment:
I think you're missing a recording for this test - there should be a `recordings/Test_AlertsManagement_SmartDetectorAlertRules_CRUD.yaml` file as well.

Can you run ` go test -count=1 -timeout 60m -run 'Test_AlertsManagement_SmartDetectorAlertRules_CRUD' -v ./internal/controllers`?

That should produce a new recording for this test and ensure it passes.

Reply:
Thanks, changes has been pushed now!